### ISSUE TITLE
React native and Vulcan

### ISSUE THREAD

**Issue by kamish24six** [VulcanJS/Vulcan#1663]
Great library! Is there a way to integrate it with a native reaction? Get data through api including authentication?

**Comment by aTylerRice** [comment]
watching this also

**Comment by FaeezBorman** [comment]
Same here! I am a newby to meteor and react, anybody have an example of vulcan + react native?

**Comment by gabimoncha** [comment]
@SachaG, it would be nice if Vulcan uses Meteor integration with Cordova. It would be even greater if Vulcan could integrate react native as its primary library for mobile apps development. What do you think about it?

**Comment by SachaG** [comment]
I agree, I personally am not working on any React Native projects myself so I don't really have the time/knowledge to improve Vulcan in that direction, but I'm definitely open to it if anybody wants to take the lead.

**Comment by zherar7ordoya** [comment]
> 
> 
> @SachaG, it would be nice if Vulcan uses Meteor integration with Cordova. It would be even greater if Vulcan could integrate react native as its primary library for mobile apps development. What do you think about it?

Agree. Maybe I'm going to add this specific request.
